.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
We should also have a section for the `Terraform` and `pip` ecosystems.

Reply:
Done